gadgets/trace_tcpconnect/README.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 title: trace tcpconnect
-sidebar_position: 230
+sidebar_position: 240
```

Comment:
The reason we're using these is because we want to push the built-in gadgets to be last on the list. I tried different ideas on https://docusaurus.io/docs/3.4.0/sidebar/autogenerated and others and I wasn't able to find a better solution than having the sidebar_position defined for each gadget.